include/ginkgo/ginkgo.hpp

```diff
@@ -125,6 +125,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/solver/multigrid.hpp>
 #include <ginkgo/core/solver/solver_base.hpp>
 #include <ginkgo/core/solver/solver_traits.hpp>
+#include <ginkgo/core/solver/triangular.hpp>
```

Comment:
only include triangular.hpp not upper_trs/lower_trs

Reply:
need to adapt the script that generates it